packages/CollabCodeEdit-Client.package/CCESession.class/instance/broadcastAddedOrModifiedMethod.in..st

```diff
@@ -0,0 +1,4 @@
+request-sending
+broadcastAddedOrModifiedMethod: aMethodSource in: aClass
```

Comment:
Strictly speaking this method is not a broadcast. the CCEClient singleton sends a single message to the server which then broadcasts it to all other clients. Because of that, this naming can be confusing when reading all other client side implemented sending methods, where no broadcast is used.

I would therefore suggest to rename the broadcast part of all methods to `sendAddedOrModifiedMethods` etc.

Reply:
Agreed. Still I would like to first have the aforementioned discussion about our architecture before I change that.